docs/customizing.rst

```diff
@@ -160,71 +160,25 @@ To customize only ``session.query``, pass the ``query_cls`` key to the
     db = SQLAlchemy(session_options={"query_cls": GetOrQuery})
 
 
-Model Metaclass
----------------
```

Comment:
Perhaps a radical approach, but I've deleted the Model Metaclass section of the customizing doc, since:
1) its main purpose was to explain table name generation, and I've now made that possible via a constructor parameter for both old and new sqlalchemy models
2) going forward, new users will hopefully be using sqlalchemy 2.x, so less need for something as esoteric as metaclass customization